service/src/main/resources/application.yml

```diff
@@ -31,7 +31,7 @@ env:
 # contains the configuration of the BPM test application. We can use that application in our
 # integration testing to make sure the application code paths are working. However, we do not
 # want it to appear in production environments.
-spring.config.import: optional:file:../config/local-properties.yml
+spring.config.import: optional:file:./generated/local-properties.yml
```

Comment:
Although this file location differs from that specified in workspace manager, there are a few reasons I think it makes sense to locate it here:

1. In practice, workspace manager doesn't appear to actually render anything into that location anyway
2. the `../config` location would put it in a folder located in `service/src/main`. I think this is better thought of as a resource
3. There was already a folder set up to be ignored by git at `service/src/main/resources/generated
4. This _is_ a generated resource